sdk/ml/azure-ai-ml/pyproject.toml

```diff
@@ -4,16 +4,8 @@ pyright = false
 type_check_samples = false
 verifytypes = false
 pylint = true
+strict_sphinx = false
```

Comment:
Sphinx false? I thought we wanted to keep sphinx active in our repos. Is this not related to that?

Reply:
I noted this in the PR description, but even though this PR fixes Sphinx errors in azure-ai-ml, azure-ai-generative, and azure-ai-resources, I was only able to turn strict_sphinx = true for the azure-ai-generative and azure-ai-resources packages because azure-ai-ml has some other errors that are currently unfixable.